functions/mult_5.R

```diff
@@ -0,0 +1,6 @@
+# Function to multiply input by 5 
+
+mult_5 <- function(x){
+  x <- x * 5
+  return(x)
```

Comment:
This is better, do this:
```suggestion
  return(x * 5)
```